### PR TITLE
Update Contributing.MD

### PR DESCRIPTION
**Pull request by gogasca**
Update expectations and best practices for both contributors and reviewers.

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/cloudml-samples/269)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,3 +27,47 @@ We are happy to see your enthusiasm. This table lists the criteria and how impor
...
+
+## Expectations
+
+### Contributors
+
+- Please read the Python Dev Guide and in particular the [Google Python Style Guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md).
+
+- Your Pull Request should be small and include a single logical change.
+
+- Smaller changes are easier to review and evaluate, and in general smaller self-contained PRs make it easier to debug or, if needed, rollback.
```
Comment (nnegrey) [inline CONTRIBUTING.md]: I think the long term vision is also to have people contribute samples. Would this kinda expectation of small changes still apply?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,3 +27,47 @@ We are happy to see your enthusiasm. This table lists the criteria and how impor
...
+
+- You should provide a sufficiently detailed description of your Pull Request.
+
+- Good descriptions make it easy to understand what the goal of the change is. Writing a good description may also make it apparent if multiple changes are being bundled in the same Pull Request that could be split.
+
+- The LGTM from the code owner ("code review") should ensure both the change intention and implementation are suitable for the codebase being changed. The review from Google team can then focus on the specificities of the Python language.
+
+- Your Pull Request should not have any pending commits (modified in Local Branch but not uploaded).
+
+- If you made changes to your Pull Request before sending it to for approval, make sure you uploaded it, too.
```
Comment (nnegrey) [inline CONTRIBUTING.md]: nit: This line and the line above seem unnecessary to me, but I'm good with them staying in too. :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (gogasca) [reply CONTRIBUTING.md]: Agree, What If I reword it to be specific for existing samples?

"- Your Pull Request for existing samples should be small and include a single logical change."

File: CONTRIBUTING.md (hunk 2 above)
Comment (gogasca) [reply CONTRIBUTING.md]: Agree

File: CONTRIBUTING.md (hunk 1 above)
Comment (nnegrey) [reply CONTRIBUTING.md]: OOO yea, that'd be great.

**Comment by puneith** [comment]
lgtm
